addons/binding/org.openhab.binding.neato/ESH-INF/thing/vacuumcleaner.xml

```diff
@@ -138,106 +166,106 @@
+	<channel-type id="is-docked">
+		<item-type>Switch</item-type>
+		<label>Is docked</label>
+		<description>Is the vacuum cleaner in the docking station?</description>
+		<state readOnly="true" pattern="%s" />
+	</channel-type>
+
+	<channel-type id="is-scheduled">
+		<item-type>Switch</item-type>
+		<label>Is scheduled enabled</label>
+		<description>True or False value if the vacuum cleaner is scheduled for cleaning.</description>
+		<state readOnly="true" pattern="%s" />
+	</channel-type>
+
+	<channel-type id="is-charging">
+		<item-type>Switch</item-type>
+		<label>Is Charging</label>
+		<description>Is the vacuum cleaner currently charging?</description>
+		<state readOnly="true" pattern="%s" />
+	</channel-type>
+
+
+	<channel-type id="error">
+		<item-type>String</item-type>
+		<label>Error</label>
+		<description>Last error message in system.</description>
+		<state readOnly="true" pattern="%s" />
+	</channel-type>
+
+
+	<channel-type id="command">
+		<item-type>String</item-type>
+		<label>Send Command</label>
+		<description>Send Commands to Vacuum Cleaner. (clean, pause, resume, stop, dock)</description>
+		<state readOnly="false" pattern="%s">
+			<options>
+				<option value="clean">Clean</option>
+				<option value="pause">Pause</option>
+				<option value="resume">Resume</option>
+				<option value="stop">Stop</option>
+				<option value="dock">Go to Dock</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="name">
+		<item-type>String</item-type>
+		<label>Vacuum Cleaner Name</label>
+		<description>Name of the vacuum cleaner represented by this Thing.</description>
+		<state readOnly="true" pattern="%s" />
+	</channel-type>
+
+	<channel-type id="cleaning-category">
+		<item-type>String</item-type>
+		<label>Cleaning Category</label>
+		<description>Current or Last category of the cleaning. Manual, Normal House Cleaning or Spot Cleaning.</description>
+		<state readOnly="true" pattern="%s">
+			<options>
+				<option value="CLEAN-CATEGORY-MANUAL">Manual Cleaning</option>
+				<option value="CLEAN-CATEGORY-SPOT">Spot Cleaning</option>
+				<option value="CLEAN-CATEGORY-HOUSE">House Cleaning</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="cleaning-mode">
+		<item-type>String</item-type>
+		<label>Cleaning Mode</label>
+		<description>Current or Last cleaning mode. Eco or Turbo.</description>
+		<state readOnly="true" pattern="%s">
+			<options>
+				<option value="CLEAN-MODE-ECO">Eco</option>
+				<option value="CLEAN-MODE-TURBO">Turbo</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="cleaning-modifier">
+		<item-type>String</item-type>
+		<label>Cleaning Modifier</label>
+		<description>Modifier of current or last cleaning. Normal or Double.</description>
+		<state readOnly="true" pattern="%s">
+			<options>
+				<option value="CLEAN-MODIFIER-NORMAL">Normal</option>
+				<option value="CLEAN-MODIFIER-DOUBLEL">Doublel</option>
```

Comment:
Typo:
<option value="CLEAN-MODIFIER-DOUBLEL">Doublel</option>
should be:
<option value="CLEAN-MODIFIER-DOUBLE">Double</option>
isn't it?